examples/features.rs

```diff
@@ -41,7 +41,7 @@ fn main() -> anyhow::Result<()> {
 
     // Now, let's define the store, and compile the module.
     let mut store = Store::new(engine);
-    let mut env = FunctionEnv::new(&mut store, ());
+    let _env = FunctionEnv::new(&mut store, ());
```

Comment:
If it's not used, we can just remove this line :)

Reply:
done